services/app-api/src/emailer/awsSES.ts

```diff
@@ -69,7 +69,9 @@ async function sendSESEmail(
     params: SendEmailRequest
 ): Promise<SendEmailResponse | SESServiceException> {
     try {
+        console.info('SendEmailCommand params: ', params)
         const command = new SendEmailCommand(params)
+        console.info('SendEmailCommand: ', command)
         return await ses.send(command)
     } catch (err) {
         console.error(JSON.stringify(err))
```

Comment:
Can you also undo our destructuring below? Just console.info(err.$$metadata) if that’s still interesting to us. The error destructuring meant that the error that got bubbled up was much less informative than it could be.